src/package/VSIXProject/TestPlatform.csproj

```diff
@@ -252,12 +252,11 @@
     <VsixSourceItem Update="$(VsixInputFileLocation)\testhost.net472.x86.exe" Ngen="true" NgenArchitecture="X86" NgenPriority="2" NgenApplication="$(ExtensionInstallationRelativeToVS)\testhost.net472.x86.exe" />
     <VsixSourceItem Update="$(VsixInputFileLocation)\testhost.net48.x86.exe" Ngen="true" NgenArchitecture="X86" NgenPriority="2" NgenApplication="$(ExtensionInstallationRelativeToVS)\testhost.net48.x86.exe" />
 
-    <!-- Arm64 is not yet supported by VsixUtil.exe and it will generate a wrong manifest. We disable temporary the ngen to avoid to enqueue arm64 binaries in the wrong architecture polluting logs with errors.-->
-    <VsixSourceItem Update="$(VsixInputFileLocation)\testhost.arm64.exe" Ngen="false" NgenArchitecture="Arm64" NgenPriority="2" NgenApplication="$(ExtensionInstallationRelativeToVS)\testhost.arm64.exe" />
-    <VsixSourceItem Update="$(VsixInputFileLocation)\testhost.net47.arm64.exe" Ngen="false" NgenArchitecture="Arm64" NgenPriority="2" NgenApplication="$(ExtensionInstallationRelativeToVS)\testhost.net47.arm64.exe" />
-    <VsixSourceItem Update="$(VsixInputFileLocation)\testhost.net471.arm64.exe" Ngen="false" NgenArchitecture="Arm64" NgenPriority="2" NgenApplication="$(ExtensionInstallationRelativeToVS)\testhost.net471.arm64.exe" />
-    <VsixSourceItem Update="$(VsixInputFileLocation)\testhost.net472.arm64.exe" Ngen="false" NgenArchitecture="Arm64" NgenPriority="2" NgenApplication="$(ExtensionInstallationRelativeToVS)\testhost.net472.arm64.exe" />
-    <VsixSourceItem Update="$(VsixInputFileLocation)\testhost.net48.arm64.exe" Ngen="false" NgenArchitecture="Arm64" NgenPriority="2" NgenApplication="$(ExtensionInstallationRelativeToVS)\testhost.net48.arm64.exe" />
+    <VsixSourceItem Update="$(VsixInputFileLocation)\testhost.arm64.exe" Ngen="true" NgenArchitecture="Arm64" NgenPriority="2" NgenApplication="$(ExtensionInstallationRelativeToVS)\testhost.arm64.exe" />
+    <VsixSourceItem Update="$(VsixInputFileLocation)\testhost.net47.arm64.exe" Ngen="true" NgenArchitecture="Arm64" NgenPriority="2" NgenApplication="$(ExtensionInstallationRelativeToVS)\testhost.net47.arm64.exe" />
+    <VsixSourceItem Update="$(VsixInputFileLocation)\testhost.net471.arm64.exe" Ngen="true" NgenArchitecture="Arm64" NgenPriority="2" NgenApplication="$(ExtensionInstallationRelativeToVS)\testhost.net471.arm64.exe" />
+    <VsixSourceItem Update="$(VsixInputFileLocation)\testhost.net472.arm64.exe" Ngen="true" NgenArchitecture="Arm64" NgenPriority="2" NgenApplication="$(ExtensionInstallationRelativeToVS)\testhost.net472.arm64.exe" />
+    <VsixSourceItem Update="$(VsixInputFileLocation)\testhost.net48.arm64.exe" Ngen="true" NgenArchitecture="Arm64" NgenPriority="2" NgenApplication="$(ExtensionInstallationRelativeToVS)\testhost.net48.arm64.exe" />
```

Comment:
As a matter of interest why do you have architecture + target framework versions of your test host? 

I would consider targeting the lowest version and then set the https://docs.microsoft.com/en-us/dotnet/api/system.appdomainsetup.targetframeworkname?view=net-6.0 to the user's target to enable quirks for that executable.

Reply:
I suppose historical reason but maybe @nohwnd has got better reason.

Reply:
Reason is that I did not know you can do it this way when I added the additional testhosts, and no-one suggested it before we do it this way.